wbooks/index.js

```diff
@@ -7,4 +7,8 @@ import { AppRegistry } from 'react-native';
 import App from './App';
 import { name as appName } from './app.json';
 
+if (__DEV__) {
+  import('./src/config/reactotronConfig');
+}
+
```

Comment:
2 things here:
* Move this to `App.tsx`:
```
import './src/config/reactotronConfig';
import App from './src/app';
...
```
* It's not necessary the `if (__DEV__)` for the import, just in the `reactotronConfig`  file is OK

Reply:
Done!